apps/data/datalayer/supervisord.conf

```diff
@@ -0,0 +1,18 @@
+[supervisord]
+nodaemon=true
+
+[program:datalayer]
+command=/datalayer
+autorestart=true
```

Comment:
Do we want it to autorestart? Usually if something in a container dies, we want the entire container to die. Now this is a multi binary container which mucks this up. Thoughts?

Reply:
Either way works. I went with this since the containers restart unless stopped and this seemed like the closest setting. If one of these services dies the whole app goes down so even though the container doesn't die we'll know pretty quick